app/src/main/java/com/googleresearch/capturesync/CameraActivity.java

```diff
@@ -482,6 +472,14 @@ private void setLeaderClientControls(boolean isLeader) {
                                 .broadcastRpc(SoftwareSyncController.METHOD_DO_PHASE_ALIGN, "");
                     });
 
+            calibrateButton.setOnClickListener(
+                    view -> {
+                        SoftwareSyncLeader softwareSyncLeader = ((SoftwareSyncLeader) softwareSyncController.softwareSync);
+                        // TODO: how to set up interaction (StereoController <-> UI in Activity) correctly?
+                        softwareSyncLeader.getStereoController().runStereoCalibration();
```

Comment:
here you have to provide some "view" interface to controller, but also remove it in activity onDestroy (if stereoController lives more than this activity)